### ISSUE TITLE
Organize import doesn't works well with baseUrl

### ISSUE THREAD

**Issue by Wykks** [buehler/typescript-hero#215]
#### System information

- TypeScript Hero version: 1.0.0

#### Description

importGroups incorrectly group workspace imports with modules imports when using `baseUrl`

##### Steps to reproduce

When have directory like "src/app/modules" (basically angular-cli setup)

1. Set `baseUrl` to `src`
2. Then add import `app/module/stuff`
3. organizeImport
4. The import is moved to modules imports

**Comment by buehler** [comment]
Hmm I really don't understand that feature. @Wykks could you elaborate the use of baseUrl and the usecase? (and maybe a code example so I can try it)

**Comment by Wykks** [comment]
In a typical angular app, you may have this structure (based on angular style guide) : 
```
/src/app/core/prefs/prefs.service
/src/app/user/user.component
/src/app/user/userPrefs/prefs.component
```

In `prefs.component`, if you need to import `prefs.service`, without `baseUrl` it will result in something like `import {PerfsService} from '../../core/prefs/prefs.service'`.
With `baseUrl` we can do this: `import {PrefsService} from 'app/core/prefs/prefs.service'`.
And it's sometimes worse than just 2 level above.

**Comment by kpaxton** [comment]
I also have this same issue. I make use of the baseUrl in the tsconfig and have many of my imports using ```app/folder1/folder2/component.component``` pattern. TypescriptHero reads those as ```Modules``` in the import groups and has them separate from the ```Workspace``` group which still has relative paths ```../folder1/folder2/component.component```. When they really should be included in the ```Workspace``` group.

I've tried to only do regex to separate out ```components``` from ```services``` and other workspace types, i.e. ```/^(app|\.|\..).*\.component$/``` but then the sorting doesn't seem to work out very well. ```app/``` ones end up being lower down in the pile with the ```./``` or ```../``` ones above. Which then TSLint complains about.

Using the baseUrl in tsConfig allows you to eliminate really long ugly relative path import statements when you have a deep folder structure. Before we used the baseUrl we had some import statements like this ```../../../../../../../../common/common-component/component.component.ts```. 

So not only including baseUrl paths in the ```Workspace``` group, but also properly sorting them against the still present relative paths.
